#! /bin/sh -e 
## install.dpatch by  <luther@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Makefiles, configure scripts and other build stuff adapted for
## DP: debian package creation

if [ $# -ne 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0
@DPATCH@
diff -ur lablgtk-1.2.6.orig/src/Makefile lablgtk-1.2.6/src/Makefile
--- lablgtk-1.2.6.orig/src/Makefile	2003-05-15 16:17:53.000000000 +0200
+++ lablgtk-1.2.6/src/Makefile	2003-10-03 13:11:44.000000000 +0200
@@ -27,7 +27,7 @@
 
 include $(CONFIG)
 
-LABLGLDIR = $(LIBDIR)/lablGL
+LABLGLDIR = $(LIBDIR)/lablgl
 
 MLLIBS = lablgtk.cma
 CLIBS = liblablgtk.a
@@ -137,6 +137,8 @@
 	rm -f testcc.ml
 
 install:
+	@$(MAKE) real-install INSTALLDIR=$(LIBDIR)/lablgtk DLLDIR=$(LIBDIR)/stublibs BINDIR=$(BINDIR)
+real-install:
 	if test -d $(INSTALLDIR); then : ; else mkdir -p $(INSTALLDIR); fi
 	cp $(ALLOBJS:.cmo=.cmi) $(THOBJS:.cmo=.cmi) $(INSTALLDIR)
 	cp -p *.mli $(INSTALLDIR)
